src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -49,6 +50,9 @@
     @Argument(fullName="createOutputBamMD5", shortName="createOutputBamMD5", doc = "If true, create a MD5 digest for any BAM/SAM/CRAM file created", optional=true)
     public boolean createOutputBamMD5 = false;
 
+    @Argument(fullName="addOutputSAMProgramRecord", shortName="addOutputSAMProgramRecord", doc = "If true, adds a PG tag to created SAM/BAM/CRAM files.", optional=true)
+    public boolean addOutputSAMProgramRecord = false;
```

Comment:
i'm fine either way. Let's leave as off for ease of testing but it's easy to flip if the need arises.